modules/nw-operator-cr.adoc

```diff
@@ -282,6 +283,14 @@ endif::ibm-z[]
 |`object`
 |Specify a configuration object for customizing network policy audit logging. If unset, the defaults audit log settings are used.
 
+|`gatewayConfig`
+|`object`
+|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway.
+
+[NOTE]
```

Comment:
@tssurya I added this note, per your comment here; https://github.com/openshift/openshift-docs/pull/40293#issuecomment-1020660174